lib/bundle/brewfile.rb

```diff
@@ -2,22 +2,22 @@
 
 module Bundle
   module Brewfile
+    ENV_KEY = "HOMEBREW_BUNDLE_FILE"
     module_function
 
     def path(dash_writes_to_stdout: false)
+      env_bundle_file = ENV.fetch(ENV_KEY, "")
+
       if ARGV.include?("--global")
+        raise "'#{ENV_KEY}' can not be specified with '--global'" unless env_bundle_file.empty?
         Pathname.new("#{ENV["HOME"]}/.Brewfile")
-      else
-        filename = ARGV.value("file")
-        if filename == "-"
-          filename = if dash_writes_to_stdout
-            "/dev/stdout"
-          else
-            "/dev/stdin"
-          end
-        end
-        filename ||= "Brewfile"
+      elsif ARGV.include?("--file")
+        filename = handle_file_value(ARGV.value("file"), dash_writes_to_stdout)
         Pathname.new(filename).expand_path(Dir.pwd)
+      elsif !env_bundle_file.empty?
```

Comment:
`elsif env_bundle_file.present?`

Reply:
```
→ bundle exec irb
irb(main):001:0> "".present?
NoMethodError: undefined method `present?' for "":String
# ...
```

Reply:
@aramprice You'll need `brew irb` as @MikeMcQuaid mentioned in https://github.com/Homebrew/homebrew-bundle/pull/502#discussion_r293581893 as it's included by Homebrew, not just bundler like you're attempting here.

Reply:
These methods are present when I run `brew irb` but tests fail when I run `bundle exec rake`. This appears to be what is [run in CI](https://github.com/Homebrew/homebrew-bundle/blob/master/azure-pipelines.yml#L13) so my expectation is that CI will not pass with `.blank?` or `.present?`

Reply:
Yes, it will need mocked in the unit tests.